### PR TITLE
draft: configuration

### PR DESCRIPTION
**Pull request by ax-ms**
# Pull Request Template

## Description

Please include a summary of the change and which issue is fixed. Include samples if adding new API, and include relevant motivation and context. List any dependencies that are required for this change.

## Type of change

Please delete options that are not relevant.

- [] Bug fix (non-breaking change which fixes an issue)
- [] New feature (non-breaking change which adds functionality)
- [] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [] This change requires a documentation update

## Closing issues

To automatically close an issue: closes #IssueNumber

### CODE REVIEW

File: Microsoft.Azure.Cosmos/src/Resource/Settings/AccountProperties.cs [hunk 1]
```diff
@@ -241,6 +241,9 @@ internal long ProvisionedDocumentStorageInMB
             }
         }
 
+        [JsonProperty(PropertyName = Constants.Properties.EnableNRegionSynchronousCommit, NullValueHandling = NullValueHandling.Ignore)]
+        public bool EnableNReginSynchronousCommit { get; set; }
```
Comment (ealsur) [inline Microsoft.Azure.Cosmos/src/Resource/Settings/AccountProperties.cs:245]: nit: 
```suggestion
        internal bool EnableNRegionSynchronousCommit { get; set; }
```

**Review by kirankumarkolli — CHANGES_REQUESTED** [review]
this branch can be used for local development and update is always sync from msdata repo.

File: Microsoft.Azure.Cosmos/src/Resource/Settings/AccountProperties.cs (hunk 1 above)
Comment (kirankumarkolli) [inline Microsoft.Azure.Cosmos/src/Resource/Settings/AccountProperties.cs:245]: Can you please offline share (teams IM) related spec?

**Comment by kirankumarkolli** [comment]
Closing the PR for now please re-open when needed

**Pull request closed without merging**